### ISSUE TITLE
Snapshot not properly code signing

### ISSUE THREAD

**Issue by postmechanical** [fastlane/snapshot#370]
I had snapshot configured and working well for a project last week and then eventually ran into the `UI Testing Failure - App accessibility isn't loaded` error because all of a sudden the app would no longer run in the simulator. The crash log shows `Code Signature Invalid` reason and `codesign -v` on the generated app gives `invalid signature (code or signature have been modified)`. Running the same UI tests from Xcode works fine and there have been no project code signing or `Snapfile` changes between the time when snapshot worked and when snapshot stopped working. I have tried all the troubleshooting steps including rebooting and `reset_simulators`. I have verified all certs and provisioning profiles are also valid and current.
